declarations/WebpackOptions.d.ts

```diff
@@ -2962,6 +2962,10 @@ export interface JavascriptParserOptions {
 	 * Enable/disable parsing "import { createRequire } from "module"" and evaluating createRequire().
 	 */
 	createRequire?: boolean | string;
+	/**
+	 * Specifies global fetchPriority for dynamic import.
+	 */
+	dynamicImportFetchPriority?: ("low" | "high" | "auto") | boolean;
```

Comment:
Yes, because you want to set it for Js, but not for CSS and verca vice and for both, so CSS will have an own property, we need to add this to CSS todo list

Reply:
Issue created and resolving.